openshift/topics/reference.adoc

```diff
@@ -58,61 +58,6 @@ pulled from the repository manager, instead of the default public repositories.
 Also, after the build is finished, you will see that the mirror is filled with
 all the dependencies that were retrieved and used during the build.
 
-
-
-A repository in Maven holds build artifacts and dependencies of various types
-(all the project jars, library jar, plugins or any other project specific
-artifacts). It also specifies locations from where to download artifacts from,
-while performing the S2I build. Besides using central repositories, it is a
-common practice for organizations to deploy a local custom repository (mirror).
-
-Benefits of using a mirror are:
-
-* Availability of a synchronized mirror, which is geographically closer and
-  faster.
-* Ability to have greater control over the repository content.
-* Possibility to share artifacts across different teams (developers, CI),
-  without the need to rely on public servers and repositories.
-* Improved build times.
-
-Often, a repository manager can serve as local cache to a mirror. Assuming that
-the repository manager is already deployed and reachable externally at
-*_pass:[http://10.0.0.1:8080/repository/internal/]_*, the S2I build can then use this
-manager by supplying the `MAVEN_MIRROR_URL` environment variable to the
-build configuration of the application as follows:
-
-. Identify the name of the build configuration to apply `MAVEN_MIRROR_URL`
-  variable against:
-+
-[source,bash,subs="attributes+,macros+"]
-----
-$ oc get bc -o name
-buildconfig/sso
-----
-. Update build configuration of `sso` with a `MAVEN_MIRROR_URL` environment variable
-+
-[source,bash,subs="attributes+,macros+"]
-----
-$ oc set env bc/sso \
-  -e MAVEN_MIRROR_URL="http://10.0.0.1:8080/repository/internal/"
-buildconfig "sso" updated
-----
-. Verify the setting
-+
-[source,bash,subs="attributes+,macros+"]
-----
-$ oc set env bc/sso --list
-# buildconfigs sso
-MAVEN_MIRROR_URL=http://10.0.0.1:8080/repository/internal/
-----
-. Schedule new build of the application
-
-NOTE: During application build, you will notice that Maven dependencies are
-pulled from the repository manager, instead of the default public repositories.
-Also, after the build is finished, you will see that the mirror is filled with
-all the dependencies that were retrieved and used during the build.
-
-[[env_vars]]
```

Comment:
This line must remain there

Reply:
@hmlnarik, @abstractj  I did add back these lines to the section.  Do they somehow need to be pushed?

=== Environment Variables

==== Information Environment Variables

Reply:
@andymunro if you added these lines back, yes, you need to push the changes again. //cc @hmlnarik

Reply:
Okay, I am not sure why this did not work before, but I added a space to the end of one line and pushed it again. Thanks, @abstractj.

Reply:
The `[[env_vars]]` line seems still be removed and needs to remain.